x-pack/test/functional/apps/maps/import_geojson/file_indexing_panel.js

```diff
@@ -103,42 +94,5 @@ export default function ({ getService, getPageObjects }) {
       const newIndexedLayerExists = await PageObjects.maps.doesLayerExist(indexName);
       expect(newIndexedLayerExists).to.be(false);
     });
-
```

Comment:
Removed these tests because they do not run on cloud. They are broken they look for geometry in `coordinates` which was changed by https://github.com/elastic/kibana/pull/120023 to `geometry`.

The behavior is validated by unit test https://github.com/elastic/kibana/blob/main/x-pack/plugins/file_upload/public/importer/geojson_importer/geojson_importer.test.js#L51